go-sdk/integration-test/e2e_test.go

```diff
@@ -657,8 +657,9 @@ func TestPostgresStorage(t *testing.T) {
 	assert.NoError(t, err)
 
 	defer func() {
-		db.Exec("DROP TABLE " + core.TreeRootsTable)
-		db.Exec("DROP TABLE " + core.NodesTablePrefix + "test_1")
+		// Table name needs to be wrapped in quotes if not it will be lowercased
+		db.Exec("DROP TABLE " + "\"" + core.TreeRootsTable + "\"")
+		db.Exec("DROP TABLE " + "\"" + core.NodesTablePrefix + "test_1\"")
```

Comment:
I think the code should check the exec commands returns no errors to avoid this happening again

Reply:
good idea